framework/src/application/node/node.js

```diff
@@ -61,31 +61,6 @@ module.exports = class Node {
 	}
 
 	async bootstrap() {
```

Comment:
Can we have `initialiseNode` to be consistent with the network

Reply:
Normally initialization meant to be constructor called. Object is initialized when called the constructor. So I kept it bootstrap explicitly with this in mind.

Reply:
then we can rename the network's `initialiseNetwork` -> `bootstrap`